packages/geo/src/locale/en.geo.json

```diff
@@ -6,6 +6,8 @@
         "layer.addToMapOutRange": "Add to map. The layer is not visible at the current scale. Please zoom in/out",
         "layer.removeFromMap": "Remove from map",
         "layer.removeFromMapOutRange": "Remove from map. The layer is not visible at the current scale. Please zoom in/out",
+        "layer.removeFromMapNotVisible": "Remove from map. The layer is visible at the current scale but is not active. Toggle the map tool to adjust layer visibility.",
+        "layer.removeFromMapVisible": "Remove from map. The layer is visible at the current scale. Toggle the map tool to adjust layer visibility.",
```

Comment:
Globally, I would remove this state...
![image](https://user-images.githubusercontent.com/7397743/222766822-e89cd287-2046-4f1c-bf26-e017cc619c69.png)

The trash can already plays this role.